docs/en/Storage.md

```diff
@@ -1,132 +1,261 @@
+use cases, but **they are equivalent** with respect to the actual data stored.
+You can:
 
-- **[Minimal](https://minimal-tenant1-minio.covid.cloud.statcan.ca):**  
-  By default, use this one. It is HDD backed storage.
-- **[Premium](https://premium-tenant1-minio.covid.cloud.statcan.ca):**  
-  Use this if you need very high read/write speeds, like for training models on
-  very large datasets.
+- Upload a file using the mounted folder on a notebook server
+- Rename that file using the web portal
+- Download that file using the `mc` tool
 
-**Publicly Available:**
+and all steps will be working on **the same file**. This lets you mix and match
+based on what is easiest for your task.
 
-- [Public (Read-Only)](https://datasets.covid.cloud.statcan.ca)
+<!-- prettier-ignore -->
+??? info "There are two different MinIO services"
+    The examples below use the `minimal-tenant1` instance of MinIO, but there is also a second instance: `premium-tenant1`.  See [Bucket Types and Access Scopes](#bucket-types-and-access-scopes) for more details.  To use `premium-tenant1` in these examples, just substitute that name in for `minimal-tenant1`.
 
-## Self-Serve
+#### MinIO Mounted Folders on a Notebook Server
 
-In any of the three self-serve options, you can create a personal bucket. To log
-in, simply use the **OpenID** option seen below.
+Automatically, all Notebook Servers have your MinIO storage mounted as
+directories. This is located in `~/minio`:
 
-![MinIO sign-in view, indicating the OpenID option](images/minio_self_serve_login.png)
+![MinIO folders mounted as Jupyter Notebook directories](images/minio_automount_folder.png)
 
-Once you are logged in, you are allowed to create a personal bucket with the
-format `firstname-lastname`.
+These folders can be used like any other - you can copy files to/from using the
+file browser, write from Python/R, etc. The only difference is that the data is
+being stored in the MinIO service rather than on a local disk (and is thus
+accessible wherever you can access your MinIO bucket, rather than just from the
+Notebook Server it is attached to like a [Disk](#disks)).
 
-![MinIO browser with personal bucket using first name, last name format (hyphenated)](images/minio_self_serve_bucket.png)
+<!-- prettier-ignore -->
+??? warn "Copying into a MinIO folder then immediately accessing the file might cause trouble"
+    There is currently a bug where files copied into a MinIO directory are sometimes not immediately available in the file browser.  They copy successfully and eventually appear in the folder system, but this may be delayed a few seconds or minutes.  If your use case needs you to copy files into a MinIO mounted folder and then access them immediately, you can still copy with the folder and then see the files using the [Web Portal](#minio-web-portal), [the mc tool](#minio-command-line-tool), or [Other S3 Compliant Methods](#other-s3-compliant-methods) to get immediate access to the file.
```

Comment:
I'd make this a `warning` rather than an `info`, but I wouldn't call it a bug. That's kinda just the nature of caching + latency on a network :man_shrugging: . I think it's worth saying though that "The file is actually uploaded in the background, so it might take a minute to sync with MinIO".

Reply:
Turns out if you just type something unused (like my typo'd `warn` instead of `warning`) you get a `note` admonition rather than an error!  Good catch - fixed